VirtualFCS/Vehicles/VehicleProfile.mo

```diff
@@ -2,40 +2,42 @@ within VirtualFCS.Vehicles;
 
 model VehicleProfile "Calculates the driving power for a vehicle that corresponds to a given speed profile."
   import Modelica.Blocks.Tables.Internal;
+  import SI = Modelica.Units.SI;
+  import NonSI = Modelica.Units.NonSI;
   type vehicle_name = enumeration(Default "Default", Mirai "Mirai", UserDefined "User Defined") annotation(
     Evaluate = true);
   parameter vehicle_name VN = VirtualFCS.Vehicles.VehicleProfile.vehicle_name.Default "Vehicle name";
   //parameter
-  Real m(unit = "kg") "mass of the vehicle";
+  SI.Mass m "mass of the vehicle";
   ////////////////////////////////////////////////////////////////////////////////////////////////////
   // *** DECLARE PARAMETERS *** //
   // Parameters of the vehicle and the air
   //parameter Real m(unit = "kg") = 1850 "Mass of the vehicle";
-  parameter Real rho_air(unit = "kg/m3") = 1.2 "Volumic mass of the air";
-  parameter Real A_front(unit = "m2") = 2.7 "Front area of the vehicle";
+  parameter SI.Density rho_air = 1.2 "Volumic mass of the air";
+  parameter SI.Area A_front = 2.7 "Front area of the vehicle";
   parameter Real C_D(unit = "1") = 0.26 "Drag coefficient";
```

Comment:
Have you considered using `Modelica.Units.SI.DimensionlessRatio` (or can `Modelica.Units.SI.Efficiency` be a candidate)?